csharp/ql/src/semmlecode.csharp.dbscheme

```diff
@@ -972,6 +975,8 @@ case @stmt.kind of
 
 @goto_any_stmt = @goto_default_stmt | @goto_case_stmt | @goto_stmt;
 
+global_stmt_block(
+  unique int id: @block_stmt ref);
```

Comment:
Do we even need this relation? Given that we extract that generated `<Main>$` method, can't the statement just be the child of that?

Reply:
Good point. I think that would work too. `global_stmt_block` is a bit more explicit than relying on the name of the container (`<Program>$.<Main>$`), but we already rely on a couple of well known names, such as `<Clone>$`. I'll make this change. I'll keep the `entry_methods` relation, which means that we still need new stats. :-(